distributed/nanny.py

```diff
@@ -314,8 +314,20 @@ def run_worker_fork(q, scheduler_addr, ncores, nanny_port,
     """
     Create a worker by forking.
     """
+    import os
+    import random
     from distributed import Worker  # pragma: no cover
+    from distributed.metrics import time
     from tornado.ioloop import IOLoop  # pragma: no cover
+    random.seed((time(), os.getpid()))
+    try:
+        import numpy as np
```

Comment:
I'm not sure, but instead of trying to import, we could just check for the module in `sys.modules`. That way, you don't load Numpy if it's not already in use.